src/actions/breakpoints/tests/breakpoints.spec.js

```diff
@@ -25,6 +28,9 @@ describe("breakpoints", () => {
       line: 2,
       sourceUrl: "http://localhost:8000/examples/a"
     };
+
+    setupHelper({ selectors: {} });
```

Comment:
lets do this in `tests/tests-setup` beforeEach

Reply:
Good shout.